oscap_report/scap_results_parser/rule_parser.py

```diff
@@ -36,8 +38,14 @@ def _get_warnings(rule):
             warnings.append(warning.text)
         return warnings
 
-    @staticmethod
-    def _get_remediations(rule):
+    def _get_remediation_code(self, fix):
+        str_fix = etree.tostring(fix).decode()
```

Comment:
I really don't like that you are dealing with XML tags in a such rude manual way (regexp, rfind, etc). ETree does now its own structure. I guess you can use `etree.tostring(root, method='text', encoding="UTF-8")` to extract the text. If you want a proper HMTL you can try `etree.tostring(root, method='html')`. See https://lxml.de/tutorial.html#serialisation.